pkg/cfn/manager/waiters.go

```diff
@@ -143,8 +143,7 @@ func (c *StackCollection) waitWithAcceptorsChangeSet(i *Stack, changesetName *st
 		if err != nil {
 			logger.Debug("describeChangesetErr=%v", err)
 		} else {
-			logger.Critical("unexpected status %q while %s", *s.Status, msg)
-			c.troubleshootStackFailureCause(i, desiredStatus)
```

Comment:
So you removed `troubleshootStackFailureCause` after all, or was it unintentional?

Reply:
That was intentional for just the changeset creation, but i added logging the reason. Its still in there for actually eexecuting the changeset (which does the actual stack update).

Reply:
👍